### PR TITLE
THE STUNSWORD

### PR DESCRIPTION
**Pull request by peeesh**
does Phillipt's stunsword donator item. Paths, sprites, everything there. Just uh, tell me how to get it in his loadout, or...do it for me

### CODE REVIEW

**Comment by peeesh** [comment]
fuck nvm conflicts, old fork. fuck my life

**Comment by peeesh** [comment]
ok it didnt fix it, wtf

**Comment by peeesh** [comment]
> ok it didnt fix it, wtf

could uh, someone figure out why

File: modular_citadel/code/modules/donatorreskins/reskinkits.dm [hunk 1]
```diff
@@ -8,6 +8,24 @@
 	icon_state = "box" //icon_state of the KIT, not the reskinned item
 	var/product = /obj/item/clothing/head/helmet/space/void/security/jenna //what it makes
 	var/list/fromitem = /obj/item/clothing/head/helmet/space/void/security //what it needs
+	afterattack(obj/O, mob/user as mob) //after being used
```
Comment (kevinz000) [inline modular_citadel/code/modules/donatorreskins/reskinkits.dm]: Relative pathing is not allowed, and don't use "as variable" unless it's unclear what variable type it is (it's pretty clear mob/user should be any /mob or subtype of /mob)

**Pull request merged**